apps/ledger-live-desktop/src/renderer/actions/settings.ts

```diff
@@ -44,6 +44,10 @@ export const setNftsViewMode = (nftsViewMode: "list" | "grid" | undefined) =>
   saveSettings({
     nftsViewMode,
   });
+export const setCollectiblesViewMode = (collectiblesViewMode: "list" | "grid") =>
```

Comment:
you can type it as : `collectiblesViewMode: Layout` (enum) or `collectiblesViewMode: LayoutKey`